connect/mirror/src/test/java/org/apache/kafka/connect/mirror/integration/MirrorConnectorsIntegrationBaseTest.java

```diff
@@ -920,13 +923,13 @@ String remoteTopicName(String topic, String clusterAlias) {
      * launch the connectors on kafka connect cluster and check if they are running
      */
     protected static void waitUntilMirrorMakerIsRunning(EmbeddedConnectCluster connectCluster,
-            List<Class<? extends Connector>> connectorClasses, MirrorMakerConfig mm2Config, 
+            List<Class<? extends Connector>> connectorClasses, MirrorMakerConfig mm2Config,
             String primary, String backup) throws InterruptedException {
         for (Class<? extends Connector> connector : connectorClasses) {
             connectCluster.configureConnector(connector.getSimpleName(), mm2Config.connectorBaseConfig(
                 new SourceAndTarget(primary, backup), connector));
         }
-        
```

Comment:
Would be nice to get ripped off these reformatting new lines changes as they are unnecessary

Reply:
Sure @OmniaGM, thanks  :)